src/tools/templates/schemas/createTemplate.ts

```diff
@@ -0,0 +1,17 @@
+import { z } from "zod";
+
+const createTemplateSchema = {
+  name: z.string().describe("Name of the template"),
+  subject: z.string().describe("Email subject line"),
+  html: z.string().describe("HTML content of the template"),
```

Comment:
This will not allow to create text only template
- but it's possible from our API and UI
- let's make html optional too and check presence inside of the tool call